### ISSUE TITLE
Library is very memory inefficient

### ISSUE THREAD

**Issue by afollestad** [amulyakhare/TextDrawable#37]
It's pretty irresponsible based on how many people use this library.

---

This library allocates Typefaces over and over again rather than caching them, resulting in a lot of memory usage. You should be caching Typefaces using something like [TypefaceHelper](https://github.com/afollestad/material-dialogs/blob/master/core/src/main/java/com/afollestad/materialdialogs/util/TypefaceHelper.java).

**Comment by markus2610** [comment]
I thought, is a caching internally integrated. With assets files is the problem or not?

**Comment by afollestad** [comment]
No, if you monitor resources while an app is running, you can see that assets are allocated multiple times.

**Comment by maxbeaudoin** [comment]
Is this resolved? How does it play with RecyclerView?
